dias/analyzers/sensitivity_analyzer.py

```diff
@@ -185,12 +185,8 @@ def run(self):
         start_time = results[0][0] if results else stop_time - self.period
 
         # Find all calibration files
-        f = self.Finder()
-        f.set_time_range(start_time, stop_time)
-        f.accept_all_global_flags()
-        f.only_corr()
-        f.filter_acqs((data_index.ArchiveInst.name == self.acq_suffix))
-        file_list = f.get_results()
+        file_list = self.find_all_staging(data_product=self.acq_suffix)
```

Comment:
For sensitvity, acq_suffix seems to be "chimestack". There is also a "correlator" attribute and not an "instrument" one. In this sense, the properties used to define the data product's names defer quite a bit from find_jumps. So we will need to modify these in ch_ansible to make them all more consistent.